core/domain/platform_parameter_registry.py

```diff
@@ -75,7 +75,8 @@ def create_platform_parameter(
                 for the platform parameter.
             is_feature: bool. True if the platform parameter is a feature flag.
             feature_stage: Enum(FeatureStages)|None. The stage of the feature,
```

Comment:
For feature flags (i.e., where 'is_feature' is True), this specifies the feature stage for that feature. For platform parameters, this value should be None.